addons/xeh/fnc_initDisplay3DEN.sqf

```diff
@@ -1,21 +1,13 @@
 #include "script_component.hpp"
 
-// fix for preInit = 1 functions not being executed when entering 3den from main menu
-[] call CBA_fnc_preInit;
+params ["_display"];
 
-// since 1.60, preInit = 1 functions aren't executed when returning from a preview either ...
-add3DENEventHandler ["OnMissionPreviewEnd", {[] call CBA_fnc_preInit}];
-
-// switching terrains in 3den will reset missionNamespace
-add3DENEventHandler ["OnTerrainNew", {[] call CBA_fnc_preInit}];
-
-private _fnc_preInitMissionConfig = {
-    {
-        if (_x select 0 == "" && {_x select 1 == "preInit"}) then {
-            [] call (_x select 2);
-        };
-    } forEach (missionConfigFile call CBA_fnc_compileEventHandlers);
+private _fnc_watchDog = {
+    if (!ISPROCESSED(missionNamespace)) then {
+        diag_log text format ["XEH: missionNamespace processed [%1]", ISPROCESSED(missionNamespace)];
```

Comment:
`ISPROCESSED(missionNamespace)` is always false, because that is the condition of the `if`.